app/controllers/api/agencies_controller.rb

```diff
@@ -15,7 +15,11 @@ def index
 
       with_event_after(@event_date)
 
-      render json: serialized_agencies
+      if @agencies.blank?
+        render json: {}
```

Comment:
I pulled the branch and when  I tried localhost:8888/api/agencies/ locally I got the results below:
...
{
"status": 404,
"error": "record_not_found",
"message": "Couldn't find Agency with 'loc_id'="
}
...

While an error message is a good idea, I think this one is not helpful.  I was thinking something like "Too many results, apply query parameters, zip_code, event_date, etc.:

Reply:
Behavior with trailing / in current production:

localhost:8888/api/agencies/ returns:
```
{
"agencies": []
}
```

localhost:8888/api/agencies/?zip_code=43123 returns expected results:

```
{
"agencies": [
{
"id": 615,
"address": "3161 E 5TH AVE",
"city": "COLUMBUS",
etc....
}
```
However with the changes in this PR, localhost:8888/api/agencies/?zip_code=43123 returns:

...
{
"status": 404,
"error": "record_not_found",
"message": "Couldn't find Agency with 'loc_id'="
}
...

So before we allowed a trailing slash before the query string and now we do not.  The error message definitely does not seem appropriate in the case of the query string.  I would have to research but I think most RFCs allow a trailing slash on the path before the query string.  I also question if the error message above is appropriate for:

localhost:8888/api/agencies/

In a REST API the intent of this this URL pattern is to return all of the agencies.  In our case, we do not allow this so the better message may be "too many agencies, filter by query params, etc.".  

Anyway I think what you have done is to say that any slash after the path is interpreted as an agency object.  If it is not valid you get a 404.  Also now this behavior is not consistent with the events and event_date APIs.